test/test_util.py

```diff
@@ -174,7 +174,17 @@ def test_parse_url_normalization(self, url, expected_normalized_url):
         ('http://foo:bar@baz@localhost/', Url('http',
                                               auth='foo:bar@baz',
                                               host='localhost',
-                                              path='/'))
+                                              path='/')),
+
+        # Unicode type (Python 2.x)
+        (u'http://foo:bar@localhost/', Url(u'http',
+                                           auth=u'foo:bar',
+                                           host=u'localhost',
+                                           path=u'/')),
+        ('http://foo:bar@localhost/', Url('http',
```

Comment:
Should these be bytes? E.g.,

```
b'http://foo:bar@localhost/'
```

Reply:
I need to do a full investigation on Python 2 and 3 to see what values used to be acceptable.

Currently on Python 3.x you can't use bytes gives a type error. On Python 2.x it looks like both unicode and str are acceptable. :)